src/librustc_mir/interpret/machine.rs

```diff
@@ -63,7 +63,6 @@ pub trait Machine<'mir, 'tcx>: Sized {
     fn mark_static_initialized<'a>(
         _mem: &mut Memory<'a, 'mir, 'tcx, Self>,
         _id: AllocId,
-        _mutability: Mutability,
```

Comment:
This argument needs to stay for miri the tool

Reply:
What does the tool use it for?

Reply:
Knowing whether the memory can be changed. The tool actually mutates statics.

Reply:
we need to do this because otherwise we deallocate the memory of the `42` in `static FOO: &u32 = &42;` after the computation is done.